relay-spans/src/span.rs

```diff
@@ -0,0 +1,350 @@
+use std::str::FromStr;
+
+use chrono::{TimeZone, Utc};
+use serde::Deserialize;
+use serde_repr::Deserialize_repr;
+
+use relay_event_schema::protocol::{Span as EventSpan, SpanId, SpanStatus, Timestamp, TraceId};
+use relay_protocol::{Annotated, Object, Value};
+
+use crate::status_codes;
+
+#[derive(Clone, Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Span {
```

Comment:
Took me a while to realize that we're implementing [this schema](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/common/v1/common.proto), might be worth linking in the doc comments.